Client/Frontend/BraveVPN/BraveVPNRegionPickerViewController.swift

```diff
@@ -0,0 +1,236 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import Shared
+import BraveShared
+import BraveUI
+import Lottie
+
+class BraveVPNRegionPickerViewController: UIViewController {
+    private let regionList: [VPNRegion]
+    
+    private var overlayView: UIView?
+    private let tableView: UITableView
+    
+    private enum Section: Int, CaseIterable {
+        case automatic = 0
+        case regionList
+    }
+    
+    deinit {
+        dispatchGroup = nil
```

Comment:
Redundant to nil this property in `deinit`? Was this meant to be a forced `leave()`?